controller/service.rst

```diff
@@ -29,10 +23,10 @@ syntax:
         // src/AppBundle/Controller/HelloController.php
         // ...
 
-        use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+        use Symfony\Component\Routing\Annotation\Route;
 
         /**
-         * @Route(service="app.hello_controller")
+         * @Route(service="AppBundle\Controller\HelloController")
```

Comment:
The `Sensio\Bundle\FrameworkExtraBundle\Configuration\Route` annotation contains the `service` option, so this won't work (see https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/Configuration/Route.php#L22). 

This case is handled in: https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/Routing/AnnotatedRouteControllerLoader.php#L41

In case of a missing service option in the route (because it's the core symfony), it will trigger this code:
```php
$route->setDefault('_controller', $class->getName().'::'.$method->getName());
```
I'm not sure if 3.3 supports this notation, but this is how routing annotation works in 3.4 without the framework extra bundle annotation. Perhaps @nicolas-grekas can confirm whether this support is in 3.3 or 3.4?

Reply:
I think 3.3